ReactWindows/ReactNative/Views/Text/ReactTextCompoundView.cs

```diff
@@ -11,7 +11,14 @@ public int GetReactTagAtPoint(UIElement reactView, Point point)
         {
             var richTextBlock = reactView.As<RichTextBlock>();
             var textPointer = richTextBlock.GetPositionFromPoint(point);
-            return textPointer.Parent.GetTag();
+            var textPointerParent = textPointer.Parent;
```

Comment:
Can this be a span or something few levels deep? In such a case textPointer.Parent is not going to be the RichTextBlock but a TextElement, so it it will need recursively walking up. Or it may that textPointer.VisualParent is easier to use?

Reply:
@gmangov - it's okay for the element to be a few levels deep. If it finds a Span parent, that parent will have been created by the `ReactSpanViewManager`, and as such will have a tag. In fact, the purpose of this `ReactTextCompoundView` is to handle exactly that case so the correct touch handler is invoked for "virtual" text elements.

This is only trying to address the one case where it finds the Paragraph (which is not correlated to the React component, but an implementation detail of using the RichTextBlock).

Reply:
I updated the comment in the source code to clarify .